components/encryption/src/manager/mod.rs

```diff
@@ -544,18 +548,13 @@ impl DataKeyManager {
     }
 
     pub fn dump_file_dict(dict_path: &str, file_path: Option<&str>) -> Result<()> {
-        let dict_file = EncryptedFile::new(Path::new(dict_path), FILE_DICT_NAME);
-        let config = MasterKeyConfig::Plaintext;
-        let backend = create_backend(&config)?;
-        let dict_bytes = dict_file.read(backend.as_ref())?;
-        let mut dict = FileDictionary::default();
-        dict.merge_from_bytes(&dict_bytes)?;
+        let (_, file_dict) = LogFile::open(dict_path, FILE_DICT_NAME, 1)?;
```

Comment:
`dump_file_dict` is used by tikv-ctl, so preferably it should be read-only (not rewriting file dict)